e2e/test/scenarios/sharing/subscriptions.cy.spec.js

```diff
@@ -66,6 +66,19 @@ describe("scenarios > dashboard > subscriptions", () => {
     cy.findByText(/Share this dashboard with people *./i);
   });
 
+  describe("sidebar toggling behavior", () => {
+    it("should allow toggling the sidebar", () => {
+      openDashboardSubscriptions();
+
+      cy.findByLabelText("subscriptions").click();
+      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
+      cy.findByText("Email it").should("not.exist");
+      cy.findByLabelText("subscriptions").click();
+      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
+      cy.findByText("Email it");
+    });
```

Comment:
We're doing this at the moment:

1. Open the subscriptions sidebar (with `openDashboardSubscriptions`)
2. Close it
3. Assert it disappeared
4. Open it again
5. Assert it appeared

Steps 4 and 5 seem redundant; we should be able to assert the sidebar is open after step 1

Reply:
Fair. I guess I was going a bit overboard with "toggling" here and closing it after it's initially open via `openDashboardSubscriptions` does test the behavior.